src/libraries/System.Formats.Asn1/tests/Reader/ReadLength.cs

```diff
@@ -27,6 +27,7 @@ public sealed class ReadLength
         [InlineData(4, 255, "0481FF")]
         [InlineData(2, 256, "02820100")]
         [InlineData(4, int.MaxValue, "04847FFFFFFF")]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/72547", typeof(PlatformDetection), nameof(PlatformDetection.IsNativeAot))]
```

Comment:
Could you please update the active issue link to https://github.com/dotnet/runtime/issues/72548 ?